commands/build.go

```diff
@@ -80,27 +79,26 @@ func runBuild(cmd *cobra.Command, args []string) {
 	}
 
 	if pullErr := PullTemplates(""); pullErr != nil {
-		log.Fatalln("Could not pull templates for OpenFaaS.", pullErr)
+		log.Fatalf("Could not pull templates for OpenFaaS.")
+		return pullErr
```

Comment:
log.Fatalf will probably cause an exit here. Let's use a combined error message with fmt.Errorf of pullErr and our custom message and return it as the error.